test/e2e/ctx/ctx.go

```diff
@@ -127,6 +128,32 @@ func (ctx TestContext) DumpNamespaceArtifacts(namespace string) error {
 	return nil
 }
 
+func (ctx TestContext) DescribeResource(command string) error {
```

Comment:
This can be largely be useful to dump out resources (or container logs) when you run into a failed test case:

```go
if CurrentGinkgoTestDescription().Failed {
	ctx.Ctx().DescribeResource(fmt.Sprintf("kubectl get operators %s -o yaml", o.GetName()))
	ctx.Ctx().DescribeResource(fmt.Sprintf("kubectl -n %s logs -l app=olm-operator --tail=50 --prefix --timestamps", *olmNamespace))
}
```